lib/wda-mac.js

```diff
@@ -143,6 +144,11 @@ class WDAMacProcess {
       return false;
     }
 
+    if (opts.webDriverAgentMacUrl) {
```

Comment:
I think we should not create `process` instance at all if `opts.webDriverAgentMacUrl` is set